_speakers/speakers.json

```diff
@@ -1,4 +1,191 @@
 [
+  {
+    "name": "Peter Munter",
+    "avatar": "https://pbs.twimg.com/profile_images/671250909/me_400x400.jpg",
+    "talks": [
+      {
+        "title": "How to do automatic WebFont setup",
+        "videoUrl": "",
+        "eventName": "2017-12-14-december-meetup"
+      }
+    ]
+  },
+  {
```

Comment:
I think we should store the speakers in a json file named the same as the markdown so that it is easier to verify and also see which events is missing to be parsed 😄 

Spot the error.

Reply:
Would splitting this data make sense? Do we know that we need the speakers data being separated? If needed, we could import this anywhere and query it as per desire.

I was thinking of adding the missing `.md` files in a follow-up PR.

Reply:
The mistake is that the speakers title was switched and I only noticed because I knew that Casper had not spoken about ReasonML. I think it is easier to edit a small json file and have the overview than it is to edit a big json where you have to edit in the middle :)